differ/differ.go

```diff
@@ -244,6 +244,26 @@ func findRuleByNameAndErrorKey(
 	return
 }
 
+// evaluateFilterExpression function tries to evaluate event filter expression
+// based on provided threshold values and actual reccomendation values
+func evaluateFilterExpression(eventFilter string,
```

Comment:
I'd use a struct instead of so many parameters

Reply:
fixed